streamer/periodconcat_node.py

```diff
@@ -0,0 +1,318 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Concatenates inputs into periods by creating a master DASH/HLS file."""
+
+import xml.etree.ElementTree as ET
+import os
+from streamer import __version__
+from streamer.node_base import NodeBase, ProcessStatus, ThreadedNodeBase
+from streamer.transcoder_node import TranscoderNode
+from streamer.packager_node import PackagerNode
+from streamer.input_configuration import Input
+from streamer.pipeline_configuration import PipelineConfig, ManifestFormat, StreamingMode
+from typing import List, Tuple, Optional
```

Comment:
Imports go in sections, each alphabatized.  Sections go: `__future__`, system, third-party, ours.  So it should be:

```py
import os
from typing import List, Tuple, Optional

from xml.etree import ElementTree

from streamer import __version__
...
```